golden_templates/upgrade-netscaler/high-availability/normal-mode/README.md

```diff
@@ -7,32 +7,56 @@
 pip install ansible==4.9.0
 ```
 2. Two NetScalers should be in High-Availability mode.
-3. Ns build Image should be present in both the NetScalers. If in case you don't have the build image available locally please refer [HERE](https://www.citrix.com/downloads/citrix-adc/) to download the Image. After you download the build, upload it to the NetScaler and Update the ansible-playbook yaml file providing the path and the file name of the build Image.
-4. Password-less SSH authentication between the controller node (system in which you are running ansiblee playbook) and the NetScalers. For more info on how to do that refer [HERE](https://github.com/citrix/citrix-adc-ansible-modules#usage)
+3. Regarding NetScaler build there are two options:
+
+    a. NetScaler build image is present in both primary and secondary NetScalers.    
+    * Update the [variables.yaml](./variables.yaml) file with `netscaler_build_location` and `netscaler_build_file_name` attributes to reflect the path and file name of the build image currently present in NetScaler, respectively
+    * Set the `want_to_copy_build` attribute's value to `no` in the same [variables.yaml](./variables.yaml) file.
+
+    b. NetScaler build image is not present in both primary and secondary NetScalers
+    * Please refer [HERE](https://www.citrix.com/downloads/citrix-adc/) to download the build image. 
+    * After downloading the build, replace the values of the attributes `want_to_copy_build` and `local_build_file_full_path_with_name` in the [variables.yaml](./variables.yaml) file with `yes` and the full path with name of the build that is now on the local system, respectively.
+
+
+4. Passwordless SSH authentication between the NetScalers and the control node(system in which the ansible-playbook is running). For additional information, see [HERE](https://github.com/citrix/citrix-adc-ansible-modules#usage).
```

Comment:
change the `usage` link to https://github.com/citrix/citrix-adc-ansible-modules/tree/887afdef75865a0ebd4bccb9c759a4b06689107a#usage